### ISSUE TITLE
id generation

### ISSUE THREAD

**Issue by EasonSun** [EasonSun/fakeClaimDetection#1]
# Class 
**Field**
claimId: dict
sourceId: dict 
stanceFeatureId: dict 
TopicId: dict 
Label 0 means true, 1 means true.
stanceFeatureLabelId: dict
claimLabelId: dict
sourceLabelId: dict 
stanceFeatureLabelId: dict
CS: numpy matrix (write out)
CT: numpy matrix (write out)
CF: numpy matrix (write out)

**Method**
getClaimId()
generateLabel()
- Use [this](http://scikit-learn.org/stable/modules/generated/sklearn.model_selection.train_test_split.html) to hold 10% as test data
writeMatrix()
writeDict()
nicelyPrint()
- print the table
? addLabelSourceId

**Usage**
Import and init this class in Yicheng's code and build CT and FC

**Comment by EasonSun** [comment]
Init can have claim id, source id, and topic id, since we append topic as a new attribute to the dataset. @Qi-Z
